storages/factory.go

```diff
@@ -54,6 +55,12 @@ func Create(ctx context.Context, destinations *viper.Viper, logEventPath string)
 		return stores, consumers
 	}
 
+	monitorKeeper, err := NewMonitorKeeper(syncServiceType, syncServiceEndpoint, connectionTimeoutSeconds)
```

Comment:
MonitorKeeper is an important part of EventNative and if we have an error we shouldn't start the server.
Please use log.Fatal() for this case. (after moving to main.go)

Reply:
done